runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/cache/KafkaCacheCursorFactory.java

```diff
@@ -876,11 +893,11 @@ public String toString()
             }
         }
 
-        private static final class Or extends KafkaFilterCondition
+        private abstract static class Or extends KafkaFilterCondition
```

Comment:
I thought the standard modifier order would be `static abstract`, no?

Reply:
We generally follow `abstract static class` as a standard and also same is followed in zilla.

Reply:
Okay thanks, I must just have it reversed in my head. 👍